### PR TITLE
Json admin

### PR DESCRIPTION
**Pull request by teleyinex**

### CODE REVIEW

**Comment by teleyinex** [comment]
This code solves issue #1348

File: test/test_activity_update.py [hunk 1]
```diff
@@ -27,7 +27,6 @@ def test_user_creation(self):
         user = UserFactory.create()
         update_feed = get_update_feed()
         err_msg = "It should be the same user"
-        assert update_feed[0]['id'] == user.id, err_msg
```
Comment (therealmarv) [inline test/test_activity_update.py]: I don't understand why this is removed?

Comment (therealmarv) [inline test/test_activity_update.py]: id not public?

File: test/test_model/test_model_user.py [hunk 2]
```diff
@@ -95,3 +95,44 @@ def test_user_errors(self):
...
+
+        user.set_password("juandiso")
+        assert public_attributes.sort() == user.public_attributes().sort()
+        data = user.to_public_json()
+        err_msg = "There are some keys that should not be public"
+        assert data.keys().sort() == public_attributes.sort(), err_msg
+        all_attributes = user.dictize().keys()
+        s = set(public_attributes)
+        private_attributes = [x for x in all_attributes if x not in s]
+        for attr in private_attributes:
```
Comment (therealmarv) [inline test/test_model/test_model_user.py]: 👍

File: test/test_activity_update.py (hunk 1 above)
Comment (teleyinex) [inline test/test_activity_update.py]: Good question. I have preferred to remove it, but yes, dunno what we should do. The main issue is that the ID of a user can be linked to a user (as that ID is stored in task_run and results) and a shortname is not. What do you think?

Comment (therealmarv) [inline test/test_activity_update.py]: ok, I only wanted to know the reason for removing it. I agree it's better this way (removing it then).

**Pull request merged**